src/basic/parse-util.c

```diff
@@ -490,3 +490,40 @@ int safe_atod(const char *s, double *ret_d) {
         *ret_d = (double) d;
         return 0;
 }
+
+int parse_fractional_part_u(const char **p, size_t digits, unsigned *res) {
+        size_t i;
+        unsigned val = 0;
+        const char *s;
+
+        s = *p;
+
+        /* accept any number of digits, strtoull is limted to 19 */
```

Comment:
Ok, but if this refers to 1/3, the number of digits after the comma is theoretically infinite, and supporting that makes no sense as we lose precision at usec level anyway, right? Also, the only user of this function passed `6` for `digit`, so I still don't see why `stroll()` doesn't suffice.